authenticators/pom.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
-  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
no need to change the license txt

Reply:
Addressed in https://github.com/wso2/samples-is/pull/338/commits/77e25c91e9c6a5bb5eca3e6b2743eed10d85ace1